packages/schematics/angular/migrations/update-vscode-recommendations/update-7.4.0.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { Rule, chain } from '@angular-devkit/schematics';
+import { updateJsonInTree } from './ast-utils';
+
+export default function(): Rule {
+  return chain([addExtensionRecommendations]);
+}
+
+const addExtensionRecommendations = updateJsonInTree(
+  '.vscode/extensions.json',
+  (json: { recommendations?: string[] }) => {
+    ['angular.ng-template', 'eg2.tslint', 'nrwl.angular-console'].forEach(
```

Comment:
Let's not introduce angular-console in the same PR, that's going to require a mit more discussion with DevRel. Open a follow-up PR adding that.

@mgechev points out that the language service doesn't work with Ivy yet but I think that's okay for v8 that we recommend an extension for most users who don't opt-in

Reply:
There's some additional work going on around the language service // cc @kyliau and @IgorMinar

Reply:
Cool, I'll remove console for this one. I also removed tslint. Would you like me to add tslint back in?

Reply:
yeah tslint belongs because the schematic creates tslint.json

Reply:
Done